pr2_description/robots/pr2.urdf.xacro

```diff
@@ -154,4 +135,36 @@
     <origin xyz=".135 0 .044" rpy="${M_PI/2} ${-32.25*M_PI/180} 0" />
   </xacro:wge100_camera_v0>
 
+  <!-- Kinect models -->
+  <xacro:arg name="kinect2" default="false"/>
+
+  <!-- Kinect1 xacro -->
+  <xacro:unless value="$(arg kinect2)">
+    <xacro:include filename="$(find pr2_description)/urdf/sensors/kinect_prosilica_camera.urdf.xacro" />
```

Comment:
Can you change the name of this to reflect that it no longer is about the prosilica camera, but only about the kinect 2?